requirements.txt

```diff
@@ -1,4 +1,8 @@
 atlalign
-atlannot
+git+https://github.com/BlueBrain/atlas-annotation@main
 atldld
-atlinter
+git+https://github.com/BlueBrain/atlas-interpolation@main
```

Comment:
IMO cloning is fine but I would pin a specific tag rather than getting the main to make things a bit more reproducible.

Reply:
Same comment as the previous one (especially now that we have expand the python versions). What do you think if we do this way:
- Handle the PR here - and fix bugs if needed (and let this the way it is for now)
- Tag new version for both `atlannot` & `atlinter`
- Create a PR tagging specific version in the `requirements.txt`

We can maybe add this point in the TO DO list to avoid forgetting about this!